packages/elements/src/collapse/index.ts

```diff
@@ -101,6 +101,13 @@ export class Collapse extends BasicElement {
   @query('ef-panel', true)
   private panel!: HTMLElement;
 
+  /**
+   * Hierarchical level of element
+   * @ignore
+   */
+  @property({ type: String, reflect: true, attribute: 'aria-level' })
```

Comment:
Why do you need it? It break Firefox AOM model. Listen for attribute changes and set a private property.

Reply:
I need this to allow developers can set a heading level to the header inside the collapse element.  Do you mean should not override all aria properties? is it conflict with AOM?

Reply:
Yes, you cannot override aria properties

Reply:
I have already removed it.